Resources/public/js/views/serverside/ez-roleserversideview.js

```diff
@@ -41,10 +44,10 @@ YUI.add('ez-roleserversideview', function (Y) {
          */
         _pickSubtree: function (e) {
             var button = e.target,
-                unsetLoading = Y.bind(this._uiUnsetAssignRoleLoading, this, button);
+                unsetLoading = Y.bind(this._uiUnsetUDWButtonLoading, this, button);
```

Comment:
@yannickroger With this PR I am not assigning roles, I am editing policy limitations. So I'd say it's a necessary change.